### PR TITLE
Add HW15

### PR DESCRIPTION
**Pull request by vladspirin**
Everything worked according to the requirements. The Linter looks okay. For more validations, only one moment can be added to that class (for example). 
Rhombus have realistic limits: 

- acute angles range approximately from 30 deg to 89 deg,
- while obtuse angles range accordingly from 91 deg to 150 deg.

### CODE REVIEW

File: lesson_15/homework15.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                raise ValueError('Sum of angel_a and b must be equal 180.')
+        # requirement 3
+        if key in ('angle_b') and value is None:
+            super().__setattr__('angle_b', self.angle_a)
+            if (180 - self.angle_a) != self.angle_b:
+                raise ValueError('Sum of angel_a and b must be equal 180.')
+        if key in ('angle_b') and value is not None and value <= 0:
+            raise ValueError(f'{key} must be a positive number')
+
+rhombus = Rhombus()
```
Comment (antohaUa) [inline lesson_15/homework15.py:56]: rhombus = Rhombus()
TypeError: __init__() missing 2 required positional arguments: 'side_a' and 'angle_a'

**Review by antohaUa — CHANGES_REQUESTED** [review]

File: lesson_15/homework15.py (hunk 1 above)
Comment (vladspirin) [reply lesson_15/homework15.py:56]: Oh, stupid mistake. Will be fixed tomorrow

**Comment by vladspirin** [comment]
I still need to learn how it happens, but that's another story. I returned my code, and everything worked well. You can check a few scenarios: 
- one side and one angle only;
- one side and two angles;
- negative or zero values;
- sum of angA and angB == 180;

**Comment by vladspirin** [comment]
@antohaUa can I merge this one?

**Review by antohaUa — APPROVED** [review]

**Pull request merged**